platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/problems/ConfigurationCacheProblems.kt

```diff
@@ -41,19 +42,11 @@ import java.io.File
 @ServiceScope(Scope.BuildTree::class)
 internal
 class ConfigurationCacheProblems(
-
-    private
-    val startParameter: ConfigurationCacheStartParameter,
-
-    private
-    val report: ConfigurationCacheReport,
-
-    private
-    val cacheKey: ConfigurationCacheKey,
-
-    private
-    val listenerManager: ListenerManager
-
+    private val startParameter: ConfigurationCacheStartParameter,
```

Comment:
In general, I actually like it better the other way since it's a more diff friendly format (consider how you would change visibility) and they are formatted as regular properties. I'd advise simply adding the new property here in the current format and proposing the style change as a separate PR.

Reply:
It seemed to me that in the majority of the Kotlin code in gradle/gradle, the primary constructors are the only place where single-line declarations are permitted _and_ used. I was just tidying up, passing by. I agree, this should be a separate PR if we want to change the style.

I think it would be better to agree as a team on the style guide for this case, make the tooling for it (automated formatting, and not only the pesky post-compilation checks), and only then enforce it.

I am curious though how the visibility changes for Java never seem to be a problem in practice. At least, I never heard this argument there.

Reply:
Kotlin members are public by default so it's more common to change visibility after the fact there. Kotlin also has internal visibility which is commonly used when tweaking visibility. That was our experience when building the Kotlin DSL anyway when the style emerged.